pdns/sdig.cc

```diff
@@ -422,7 +422,7 @@ try {
     Socket sock(dest.sin4.sin_family, SOCK_STREAM);
     sock.setNonBlocking();
     setTCPNoDelay(sock.getHandle()); // disable NAGLE, which does not play nicely with delayed ACKs
-    TCPIOHandler handler(subjectName, false, sock.releaseHandle(), timeout, std::move(tlsCtx));
+    TCPIOHandler handler(subjectName, false, sock.releaseHandle(), timeout, tlsCtx);
```

Comment:
Coverity does not seem to agree, see previous commit: https://github.com/PowerDNS/pdns/commit/3b45a4347c8af1b28e6871597221eb906c143a29

Reply:
Coverity was right at the time, but https://github.com/PowerDNS/pdns/commit/224085cccd7d7f52ceac3bd1f0ec8e4cd262948d changed the constructor.

Reply:
Fixed.